schema-lending.graphql

```diff
@@ -332,9 +256,18 @@ type Market @entity {
   " Total borrow volume in USD (i.e. total loan origination ) "
   totalVolumeUSD: BigDecimal!
 
+  " Current balance of all deposited assets (not historical cumulative), in USD "
+  totalDepositUSD: BigDecimal!
+
+  " Current balance of all borrowed assets (not historical cumulative), in USD. In the case of CDPs, this will be all minted assets "
+  totalBorrowUSD: BigDecimal!
+
   " Amount of input tokens in the market. The ordering should be the same as the market's `inputTokens` field. "
   inputTokenBalances: [BigInt!]!
 
+  " Price per share of input token in USD "
+  inputTokenPricesUSD: [BigDecimal!]!
+
   " Total supply of output token "
   outputTokenSupply: BigInt!
```

Comment:
same here i guess. my understanding for compound is the number of cTokens outstanding. Once it is redeemed the cToken supply is decreased in the contract. And I misunderstood this at first